source/sdk/lang/Exception.ooc

```diff
@@ -47,6 +49,18 @@ _pushStackFrame: inline func -> StackFrame {
     buf
 }
 
+_markStackFrame: inline func (frame: StackFrame) {
+  stackFramesToDelete push(frame)
+}
+
+_cleanupStackFrames: inline func {
+  while (stackFramesToDelete empty?() == false) {
+    frame := stackFramesToDelete pop() as StackFrame
+    if (frame)
+      gc_free(frame as Void*)
```

Comment:
I wonder if it's possible to define a `free` function on the `StackFrame` cover so you could call `frame free()` instead...